### DISCUSSION TITLE
warm up error

### DISCUSSION THREAD

**Discussion by rickyurvinauc** [asreview/asreview#1750] [Troubleshooting (report errors in the "Issues" tracker)]
I have an error when I click in the last step in project creating
This is the error

  File "FoodDetectionReview/asreview/asreview/__main__.py", line 53, in main
    entry = base_entries_internal[sys.argv[1]].load()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/metadata/__init__.py", line 204, in load
    return functools.reduce(getattr, attrs, module)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'asreview.webapp.entry_points.run_model' has no attribute 'run_model_entry_point'

**Comment by rickyurvinauc** [comment]
[undefined-2.asreview.zip](https://github.com/asreview/asreview/files/15409065/undefined-2.asreview.zip)
 is my project and in the app stay on
 
![image](https://github.com/asreview/asreview/assets/163784827/fb94d3f2-a15d-488d-99bc-d78a9217cea7)

**Reply by Rensvandeschoot** [reply]
Thank you for sharing your file! I was able to open the project and train a model, so there is no issue with your data! (if you send an email to asreview@uu.nl  I can send you the project file)

But it should also work on your device... What version of ASReview are you using?


![image](https://github.com/asreview/asreview/assets/36502709/748b24b2-cfcc-4133-a6da-46bd636c9ab0)
